pkg/ctl/cmdutils/cmdutils.go

```diff
@@ -103,6 +104,12 @@ func GetNameArg(args []string) string {
 	return ""
 }
 
+// IsValidNameArg checks whether the name contains invalid characters
+func IsValidNameArg(name string) bool {
```

Comment:
Since we always call the func like `!cmdutils.IsValidNameArg()` shall we inverse the return value and make it `isInvalidNameArg`?

Reply:
changed to the suggested name